aten/src/ATen/native/cpu/zmath.h

```diff
@@ -8,32 +8,6 @@
 
 namespace at { namespace native {
 inline namespace CPU_CAPABILITY {
-
-template <typename SCALAR_TYPE, typename VALUE_TYPE=SCALAR_TYPE>
-inline VALUE_TYPE zabs (SCALAR_TYPE z) {
-  return z;
-}
-
-template<>
-inline c10::complex<float> zabs <c10::complex<float>> (c10::complex<float> z) {
```

Comment:
instantiation which returns the same dtype for `complex` is not used any-where in the codebase.

other instantiation which returns real number is same as `std::abs`

Have updated relevant call sites with std::abs where it made sense.

Reply:
Really nice cleanup